misc/nsis/x86.nsi

```diff
@@ -0,0 +1,66 @@
+Name "qutebrowser"
+OutFile "qutebrowser-x86.exe"
```

Comment:
I wonder... since the x86 and x64 differences are only three lines or so, wouldn't it make more sense to use [`!ifdef`](http://nsis.sourceforge.net/Reference/!ifdef) and starting `makensis` with [`/DX64`](http://nsis.sourceforge.net/Docs/Chapter3.html) or so for the 64bit one?